src/NLog.Targets.Syslog/MessageCreation/Rfc3164.cs

```diff
@@ -58,7 +57,7 @@ private void AppendHeader(ByteArray buffer, LogEventInfo logEvent)
         {
             if (!outputHeader)
                 return;
-            var timestamp = string.Format(CultureInfo.InvariantCulture, TimestampFormat, logEvent.TimeStamp);
+            var timestamp = timestampLayout.Render(logEvent);
```

Comment:
If you try running one of the test apps you will see that this is not equivalent to the line it replaces